index.bs

```diff
@@ -13616,6 +13623,189 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 </div>
 
+## The autofill Module ## {#module-autofill}
+
+The <dfn export for=modules>autofill module</dfn> contains functionality for triggering autofill.
+
+### Definition ### {#module-autofill-definition}
+
+{^remote end definition^}
+
+<pre class="cddl" data-cddl-module="remote-cddl">
+
+AutofillCommand = (
+  autofill.Trigger
+)
+</pre>
+
+### Commands ### {#module-autofill-commands}
+
+#### The autofill.trigger Command ####  {#command-autofill-trigger}
+
+The <dfn export for=commands>autofill.trigger</dfn> command triggers autofill
+on a particular form field using the provided fields.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl" data-cddl-module="remote-cddl">
+      autofill.Trigger = (
+        method: "autofill.trigger",
+        params: autofill.TriggerParameters
+      )
+
+      autofill.TriggerParameters = {
+        context: browsingContext.BrowsingContext,
+        element: script.SharedReference,
+        ? card: autofill.Card,
+        ? address: autofill.Address
+      }
+
+      autofill.Field = {
+        name: autofill.FieldName,
+        value: text
+      }
+
+      autofill.Card = {
+        number: text,
+        name: text,
+        expiryMonth: text,
+        expiryYear: text,
+        cvc: text
+      }
+
+      autofill.Address = {
```

Comment:
The fields that an address consists of depends on the locale of the website, form, and user agent. How do you plan to deal with this? I'm thinking that Japanese has both romanized and traditional address formats, but also Hindi and others.